MonoGame.Framework/Graphics/Texture2D.cs

```diff
@@ -146,6 +149,18 @@ private static void ConvertToABGR(int pixelHeight, int pixelWidth, int[] pixels)
                 pixels[i] = (int)((pixel & 0xFF00FF00) | ((pixel & 0x00FF0000) >> 16) | ((pixel & 0x000000FF) << 16));
             }
         }
+
+        /// <summary>
+        /// Returns a code unique to this <see cref="Texture2D"/> instance, used for sorting
+        /// textures with <see cref="SpriteSortMode.Texture"/>.
+        /// </summary>
+        internal int ComparisonCode
+        {
+            get { return _comparisonCode; }
+        }
+
+        private int _comparisonCode;
+        private static int _nextComparisonCode;
```

Comment:
This should be moved to `Texture` so it is available to all derived texture classes.